src/Tribe/Venue.php

```diff
@@ -495,16 +495,19 @@ public function update( $venue_id, $data ) {
 		                      ) );
 
 		if ( count( $args ) > 1 ) {
+			$tag = 'save_post_' . Tribe__Events__Main::VENUE_POST_TYPE;
+			remove_action( $tag, array( tribe( 'tec.main' ), 'save_venue_data' ), 16 );
```

Comment:
not in this case: we'd need to update that on main too and I do not want to make that leap now.